launcher/res/Strings.ja-jp.xaml

```diff
@@ -0,0 +1,173 @@
+    <sys:String x:Key="Strings.ProfileLinkFormat">Windower ({0})</sys:String>
+    <sys:String x:Key="Strings.CopyProfile">コピーを作成</sys:String>
+    <sys:String x:Key="Strings.ProfileCopyNameFormat">{0} (Copy {1})</sys:String>
+    <sys:String x:Key="Strings.ClientNotFound">WindowerがFinal Fantasy XIの場所を特定できません</sys:String>
+    <sys:String x:Key="Strings.ClientNotFoundDescription">その機能を利用するにはFinal Fantasy XIのインストールが必要です</sys:String>
+    <sys:String x:Key="Strings.DirectPlayPrompt">DirectPlayがインストールされていません</sys:String>
+    <sys:String x:Key="Strings.DirectPlayDescription">Final Fantasy XIはDirectPlayを利用しています。DirectPlayはDirectX 8.1のオプショナルコンポーネントのため、Windows 8以降ではデフォルトでインストールされていません。DirectPlayをインストールしますか？</sys:String>
+    <sys:String x:Key="Strings.Install">インストール</sys:String>
+    <sys:String x:Key="Strings.Exit">終了</sys:String>
+    <sys:String x:Key="Strings.Crashed">Windowerは不具合によりクラッシュしました</sys:String>
+    <sys:String x:Key="Strings.CrashReporterPreparing">クラッシュに関する情報を収集しています</sys:String>
+    <sys:String x:Key="Strings.CrashReporterCheckingReport">この問題が既に修正されているか調べています</sys:String>
+    <sys:String x:Key="Strings.CrashReporterSigningIn">サインイン</sys:String>
+    <sys:String x:Key="Strings.CrashReporterVerifying">アイデンティティを確認中</sys:String>
+    <sys:String x:Key="Strings.CrashReporterCreatingGist">クラッシュレポートのgistを作成中</sys:String>
+    <sys:String x:Key="Strings.CrashReporterCloning">クラッシュレポートの送信準備中</sys:String>
+    <sys:String x:Key="Strings.CrashReporterEncrypting">クラッシュレポートのデータを暗号化しています</sys:String>
+    <sys:String x:Key="Strings.CrashReporterStaging">クラッシュレポートの送信準備中</sys:String>
+    <sys:String x:Key="Strings.CrashReporterPackaging">クラッシュレポートをパッケージングしています</sys:String>
+    <sys:String x:Key="Strings.CrashReporterTransferring" xml:space="preserve">クラッシュレポートをGitHubに送信中&#xD;&#xA;(この操作が完了するまで少し時間がかかります)</sys:String>
+    <sys:String x:Key="Strings.CrashReporterFetchingGistDetails">gistの詳細を取得中</sys:String>
+    <sys:String x:Key="Strings.CrashReporterCreatingIssue">このクラッシュレポートに関するチケットを作成しています</sys:String>
+    <sys:String x:Key="Strings.CrashReporterCreatingComment">このクラッシュレポートに関するチケットにコメント</sys:String>
+    <sys:String x:Key="Strings.CrashReporterSubmitPrompt">Windower開発チームにクラッシュレポートを送信しますか？</sys:String>
+    <sys:String x:Key="Strings.CrashReporterSubmitDisclaimerEncrypted" xml:space="preserve">クラッシュレポートが生成されました。このレポートはWindower開発チームが問題を修正する一助となります。このクラッシュレポートには不具合発生時のWindowerとOSの状態に関する情報が含まれています。 収拾された情報は不具合修正のためだけに利用され、送信される前に暗号化されるので第三者が情報を読むことはありません。&#xD;&#xA;&#xD;&#xA;クラッシュレポートの送信にはGitHubアカウントが必要です。レポートはあなたのGitHubアカウントのgistとして送信され、チケットに関連付けられます。</sys:String>
+    <sys:String x:Key="Strings.CrashReporterSubmitDisclaimer" xml:space="preserve">クラッシュレポートが生成されました。このレポートはWindower開発チームが問題を修正する一助となります。このクラッシュレポートには不具合発生時のWindowerとOSの状態に関する情報が含まれています。 収拾された情報は不具合修正のためだけに利用され、送信される前に暗号化されるので第三者が情報を読むことはありません。&#xD;&#xA;&#xD;&#xA;クラッシュレポートの送信にはGitHubアカウントが必要です。レポートはあなたのGitHubアカウントのgistとして送信され、チケットに関連付けられます。</sys:String>
```

Comment:
This seems to be the same as the entry before, `CrashReporterSubmitDisclaimerEncrypted`. Only the entry before should mention the encryption, this one should not.

Reply:
I will fix it.